backend/user-service/src/controllers/authController.ts

```diff
@@ -101,30 +129,162 @@ export const logIn: RequestHandler[] = [
       return;
     }
 
+    // const { password: _, ...userWithoutPassword } = user;
+    
+    try {
+      // const accessToken = await generateAccessToken(userWithoutPassword);
+      // const refreshToken = await generateRefreshToken(userWithoutPassword);
+      // storedRefreshTokens.push(refreshToken);
+
+      // res.cookie("accessToken", accessToken, { httpOnly: true, secure: false });
+      // res.cookie("refreshToken", refreshToken, { httpOnly: true, secure: false });
+      
+      // return res.status(200).json({
+      //   message: `${user.username} has been authenticated`,
+      //   accessToken,
+      //   refreshToken,
+      // });
+      return await generateBothTokens(req, res, user);
```

Comment:
there is a way to attach user to the request and make this function middleware
see #40